var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -12,13 +12,16 @@ class Paraview(CMakePackage):
     visualization application."""
 
     homepage = 'http://www.paraview.org'
-    url      = "http://www.paraview.org/files/v5.3/ParaView-v5.3.0.tar.gz"
-    _urlfmt  = 'http://www.paraview.org/files/v{0}/ParaView-v{1}{2}.tar.gz'
+    url      = "http://www.paraview.org/files/v5.6/ParaView-v5.6.2.tar.xz"
+    list_url = "http://www.paraview.org/files"
+    list_depth = 1
+    _urlfmt  = 'http://www.paraview.org/files/v{0}/ParaView-v{1}{2}.tar.xz'
```

Comment:
Bumping to `5.6.2` it seems will require a more complex url handler since there was a shift from `tar.gz` to `tar.xz`.  Just changing the url like this breaks downloads for anything before `5.5.x` when the `tar.xz` became available and breaks the checksums for everything before `5.6.2` since they were based on the `tar.gz`

Reply:
@danlipsa